nixos/tests/ejabberd.nix

```diff
@@ -0,0 +1,259 @@
+import ./make-test.nix {
```

Comment:
Would you please list yourself as the maintainer of this test?

Reply:
Done.